src/components/Notifications/Messages.tsx

```diff
@@ -48,6 +48,10 @@ export const onlyWhiteSpaces = ()=>{
     return `Назва не може містити лише порожні символи`
 };
 
+export const inputOnlyWhiteSpaces = ()=>{
+    return `Поле не може містити лише порожні символи`
+};
```

Comment:
Don't you think that the previous function at lines 47-49 is actually the same?

Reply:
you are right, I can delete first one and use only second

Reply:
Done